test/performance/run-test.sh

```diff
@@ -82,29 +82,28 @@ for item in "${EXPERIMENTS[@]}"; do
         yq -i e ".spec.resources[1].flavors[0].quota.min=\"$memory\"" prerequisites/cluster-queue.yaml
         kubectl apply -f prerequisites/cluster-queue.yaml
     fi
-    "$CL2_HOME_DIR/$CL2_BINARY_NAME" \
+    "$CL2_HOME_DIR"/"$CL2_BINARY_NAME" \
         --testconfig=config.yaml \
         --enable-prometheus-server=true \
         --provider="$PROVIDER" \
         --v=2 --prometheus-scrape-metrics-server=true \
         --prometheus-scrape-kube-state-metrics=true \
         --report-dir="$experiment_dir"
-    echo "Experiment finished. Extracting results from the report..."
-    {
+    echo "Experiment finished. Extracting results from the report..." {
         echo -ne "$CL2_SMALL_JOBS $CL2_MEDIUM_JOBS $CL2_LARGE_JOBS $CL2_JOB_RUNNING_TIME $CL2_TEST_TIMEOUT $cores $memory,"
-        echo -ne "$(jq '.dataItems[] | select(.labels.Metric=="create_to_start").data.Perc50' "$experiment_dir"/JobLifecycleLatency*.json),"
-        echo -ne "$(jq '.dataItems[] | select(.labels.Metric=="create_to_start").data.Perc90' "$experiment_dir"/JobLifecycleLatency*.json),"
-        echo -ne "$(jq '.dataItems[] | select(.labels.Metric=="start_to_complete").data.Perc50' "$experiment_dir"/JobLifecycleLatency*.json),"
-        echo -ne "$(jq '.dataItems[] | select(.labels.Metric=="start_to_complete").data.Perc90' "$experiment_dir"/JobLifecycleLatency*.json),"
-        echo -ne "$(jq '.dataItems[0].data.max_job_throughput' "$experiment_dir"/GenericPrometheusQuery*.json),"
-        echo -ne "$(jq '.dataItems[0].data.total_jobs_scheduled' "$experiment_dir"/GenericPrometheusQuery*.json),"
-        echo -ne "$(jq '.dataItems[0].data.total_pods_scheduled' "$experiment_dir"/GenericPrometheusQuery*.json),"
-        echo -ne "$(jq '.dataItems[0].data.job_performance' "$experiment_dir"/Timer*.json)",
-        echo -ne "$(jq '.dataItems[0].data.avg_pod_waiting_time' "$experiment_dir"/GenericPrometheusQuery*.json),"
-        echo -ne "$(jq '.dataItems[0].data.perc_90_pod_waiting_time' "$experiment_dir"/GenericPrometheusQuery*.json),"
-        echo -ne "$(jq '.dataItems[0].data.avg_pod_running_time' "$experiment_dir"/GenericPrometheusQuery*.json),"
-        jq '.dataItems[0].data.perc_90_pod_completion_time' "$experiment_dir"/GenericPrometheusQuery*.json
-    } >>"$report_dir_name/summary.csv"
+        echo -ne "$(jq '.dataItems[] | select(.labels.Metric=="create_to_start").data.Perc50' "$experiment_dir/JobLifecycleLatency*.json),"
```

Comment:
this change looks odd. What is closing the `"`?

Reply:
On a closer look, this looks wrong. For now, I have reverted the changes in this file. I'll handle all the shell check errors in separate PR(s) for easy review, and I can get more done in smaller chunks faster.